java-2016/homework-9/IOhomework/src/AthleteEvaluation.java

```diff
@@ -0,0 +1,106 @@
+import java.util.*;
+
+
+/**
+ * Created by Turian Ovidiu.
+ * This class is responsible for calculation the ranking.
+ */
+public class AthleteEvaluation {
+
+
+    public AthleteEvaluation() {
+    }
+
+
+    /**
+     * This method takes a list of Athletes and will return an list sorted by ski time result after calculation the shots.
+     * @param athletes the list of Athletes.
+     * @return list of Athletes sorted by ski time result after calculation the shots.
+     */
+    public List<Athlete> rankingAthletes(List<Athlete> athletes) {
```

Comment:
name should be getAthletesRanking 
Basically rule is that whenever you return something that should be reflected in the name. So name might be get, parse, etc